oasislmf/execution/bin.py

```diff
@@ -247,7 +263,13 @@ def prepare_run_inputs(analysis_settings, run_dir, ri=False):
 
         # Prepare occurrence / returnperiod depending on output calcs selected
         if _calc_selected(analysis_settings, 'lec_output'):
-            _prepare_input_bin(run_dir, 'returnperiods', model_settings, ri=ri)
+            if 'return_periods' in analysis_settings:
```

Comment:
In order to avoid failure if there is something like this in the settings
"return_periods": {} 
I would use:
if analysis_settings.get('return_periods'):
It feels safer

Reply:
ah yes fair point, I'll switch it over